packages/python/plotly/requirements.txt

```diff
@@ -10,3 +10,6 @@ six==1.8.0
 
 ## retrying requests ##
 retrying==1.3.3
+
+## Allow pathlib2 as pathlib substitute for old Python versions ##
```

Comment:
Let's remove this, as we're dropping support for 2.7 and 3.5 in the next Plotly.py release.